tools/internal_ci/linux/grpc_xds_k8s_xlang.sh

```diff
@@ -109,15 +120,15 @@ main() {
   local failed_tests=0
   local successful_string
   local failed_string
-  # Run tests
-  for TAG in ${VERSION_TAG}
+  # Run cross lang tests: for given cross lang versions
+  for TAG in ${XLANG_VERSIONS}
   do
     for CLANG in ${CLIENT_LANG}
     do
     for SLANG in ${SERVER_LANG}
```

Comment:
We should change it to 

`for CLIENT_LANG in ${CLIENT_LANGS}`
and
`for SERVER_LANG in ${SERVER_LANGS}`

Reply:
done